Swift/ReplicatorConfiguration.swift

```diff
@@ -384,13 +384,21 @@ public struct ReplicatorConfiguration {
     
     // MARK: Internal
     
-    var defaultCollectionConfig: CollectionConfiguration {
+    var defaultCollectionConfig: CollectionConfiguration? {
         guard let col = try? self.database.defaultCollection() else {
             fatalError("Default collection is missing!")
         }
         
-        guard let colConfig = self.collectionConfigs[col] else {
-            fatalError("default collection config missing")
+        if let config = self.collectionConfigs[col] {
```

Comment:
I don't see the implementation that makes Collection a Hashable. Is it not required?

Reply:
it is already hashable. 
https://github.com/couchbase/couchbase-lite-ios/blob/master/Swift/Collection.swift#L286-L294